options/posix/generic/unistd-stubs.cpp

```diff
@@ -194,14 +201,29 @@ int faccessat(int dirfd, const char *pathname, int mode, int flags) {
 }
 
 int fchown(int fd, uid_t uid, gid_t gid) {
-	mlibc::infoLogger() << "\e[31mmlibc: fchown() is not implemented correctly\e[39m"
-			<< frg::endlog;
+	if(!mlibc::sys_fchownat) {
+		MLIBC_MISSING_SYSDEP();
+		errno = ENOSYS;
+		return -1;
+	}
+	if(int e = mlibc::sys_fchownat(fd, nullptr, uid, gid, AT_EMPTY_PATH); e) {
```

Comment:
The description states that the path name needs to be an empty string (and not NULL) for this to work.